src/Windows/Avalonia.Win32/WindowImpl.cs

```diff
@@ -148,7 +148,9 @@ public IRenderer CreateRenderer(IRenderRoot root)
             if (customRendererFactory != null)
                 return customRendererFactory.Create(root, loop);
 
-            return Win32Platform.UseDeferredRendering ? (IRenderer)new DeferredRenderer(root, loop) : new ImmediateRenderer(root);
+            return Win32Platform.UseDeferredRendering
+                ? (IRenderer)new DeferredRenderer(root, loop)
```

Comment:
Nit: the `?` in these sort of statements is usually put at the end of the condition for multi-line statements.

Reply:
```
condition
     ? left
     : right;
```
is a pretty common pattern for multi-line expressions.